x/supply/internal/types/supply.go

```diff
@@ -3,7 +3,7 @@ package types
 import (
 	"fmt"
 
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
this is unnecessary - possibly leftover introduced by IntelliJ

Reply:
it's from `make format`

Reply:
Weird, it does not do it for me (I've just regenerated my whole GOPATH).

Reply:
This shouldn't happen -- should be undone.

Reply:
everywhere this package is used we are doing `yaml "gopkg.in/yaml.v2"`. This will break golangci and I'm opposed to using `// nolint:goimports`

Reply:
go imports makes this happen whenever the import path is not the same as the package name. I think it's fine to leave this